recipe/meta.yaml

```diff
@@ -12,20 +12,17 @@ source:
     - no-console.patch
 
 build:
-  number: 2
-  features:
-    - vc9  # [win and py27]
-    - vc14  # [win and py>=35]
-  skip: true  # [win and py36]
+  number: 4
 
 requirements:
   build:
-    - toolchain
+    - {{ compiler('cxx') }}
     - cmake  # [win]
 
 test:
   requires:
-    - python {{ os.environ.get('PY_VER', '') + '*' }}  # [win]
+    - python 3.6  # [not (win and vc<14)]
```

Comment:
I added this because otherwise it was making a Python version matrix (which it really shouldn't be...). Not sure why that didn't happen in https://github.com/conda-forge/muparser-feedstock/pull/16....